relation.go

```diff
@@ -27,19 +27,22 @@ type relations struct {
 type relation struct {
 	Id_        int        `yaml:"id"`
 	Key_       string     `yaml:"key"`
+	Status_    string     `yaml:"status,omitempty"`
```

Comment:
I don'think this should be omitempty. We don't need to marshal v1, we just need to unmarshal it; and it doesn't matter if it's omitempty either way for unmarshalling. Marking this omitempty suggests that a v2 relation might have an empty status.